src/gsumm.c

```diff
@@ -940,8 +940,6 @@ SEXP gmedian(SEXP x, SEXP narmArg) {
 
 SEXP glast(SEXP x) {
 
-  if (!isVectorAtomic(x)) error("GForce tail can only be applied to columns, not .SD or similar. To get tail of all items in a list such as .SD, either add the prefix utils::tail(.SD) or turn off GForce optimization using options(datatable.optimize=1).");
```

Comment:
There was a contradiction in both `glast` & `gfirst` between this error branch and the `VECSXP` branch below. I decided to try and remove this error & see what happens, and all the tests pass, so I opted to resolve the contradiction by eliminating this branch.